test/integration/manifests/load/noop-deployment-windows.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1

```

Comment:
lets not name directory as load.. i think these files can be used by other windows pipeline as well